vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -458,8 +458,8 @@ export default class BaseAddress extends Mixins(ValidationMixin, CountriesProvin
   }
 }
 
-.pre-wrap {
-  white-space: pre-wrap;
+.pre-line {
+  white-space: pre-line;
```

Comment:
Did you test this in Edge, Chrome, FF? Safari if you're able? (Maybe check IE11 as well though we don't try hard to support it.)

Reply:
I tested in Edge, IE, Chrome and FF.

Reply:
Was it broken before, and fixed now?

Reply:
Also, what do the addresses long like with long lines?

You can paste screen shots here to show your testing (or paste in ticket).

Reply:
in FF, before change:
![image](https://user-images.githubusercontent.com/54647458/120019146-ed2ec200-bf9c-11eb-83de-8b2715c5848a.png)

Reply:
after code change in FF:
![image](https://user-images.githubusercontent.com/54647458/120019332-2e26d680-bf9d-11eb-846f-7c3ce6054d1b.png)

Reply:
OK, the fix above looks good. But it seems to undo a previous fix, so I'm wondering if this actually breaks something else? Like long address lines maybe?

Reply:
What does long address lines mean? The max length of the address line is 50 and it is good -- is it sufficient?

Reply:
In the example above, it looks like 50 chars will easily fit on the line, but in a different example (or possibly on a narrower screen), 50 chars might not fit on the line. In that case, the address needs to wrap correctly.

I just had a look at https://www.w3schools.com/cssref/pr_text_white-space.asp ... Both the previous and new CSS properties appear to support wrapping, so you're probably OK, but please verify it.

As a general concept, since there are no unit tests here to verify whitespace behaviour, you will have to manually verify the different scenarios. I can think of the following (there may  be more):
- does long address line wrap correctly?
- is whitespace collapsed or preserved?

Reply:
Comparing pre-wrap and pre-line, they all wrap text to fit the container. pre-wrap does NOT collapse whitespace while pre-line collapse extra whitespace to one. To my understanding, replace pre-wrap with pre-line should fix the bug and no other issues related.